CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changelog
 
 ### Features
 
+- Improved scrollbar look and feel on wallets list ([PR 1475](https://github.com/input-output-hk/daedalus/pull/1475))
```

Comment:
@yakovkaravelov this is WRONG. We are NOT releasing this with 0.14.0 version of Daedalus. It has to be moved to a new `vNext` section.

Reply:
@nikolaglumac Fixed.